docs/guides/chip_tool_guide.md

```diff
@@ -913,28 +952,35 @@ will be required by the second Matter admin to join Matter device to its fabric.
 [1663675289.149445][56387:56392] CHIP:CTL: SetupQRCode: [MT:4CT91AFN00YHEE7E300]
 ```
 
-To commission the Matter device to a new fabric, you need to run another
-instance of CHIP Tool, using the following command pattern:
+#### Step 4: Commission the Matter device to a new fabric
 
-```
-$ ./chip-tool pairing code <payload> <node_id> --commissioner-name <commissioner_name>
-```
+Complete the following steps:
 
-In this command:
+1. Open another instance of the CHIP Tool.
+1. In the new instance of the CHIP Tool, commission the Matter device to a new
```

Comment:
There is a mistake in enumerating: 1->2.

Reply:
This is Markdown formatting for steps that renders them with correct order when rendering the rich diff or preview.